main/src/main/scala/sbt/internal/CommandExchange.scala

```diff
@@ -376,13 +377,14 @@ private[sbt] final class CommandExchange {
 
   private[sbt] def setExec(exec: Option[Exec]): Unit = currentExecRef.set(exec.orNull)
 
-  def prompt(event: ConsolePromptEvent): Unit = {
+  def prompt(event: ConsolePromptEvent): Unit =
     currentExecRef.set(null)
     channels.foreach {
       case c if ContinuousCommands.isInWatch(lastState.get, c) =>
-      case c                                                   => c.prompt(event)
+      case c =>
+        if c.isPaused then ()
+        else c.prompt(event)
```

Comment:
Also if the channel is paused we should not show the `sbt server is running ...` message:

![image](https://github.com/user-attachments/assets/ec955eaf-bc44-4f1d-a4b2-1b6d296893ab)

Reply:
Nor this one:

![image](https://github.com/user-attachments/assets/7e2788f5-1314-4b89-89be-7ea5c6e87198)